### ISSUE TITLE
User home dir shortcuts don't work

### ISSUE THREAD

**Issue by ghostravenstorm** [gkngkc/UnityStandaloneFileBrowser#120]
Shortcuts to the user's home dir like `%userprofile%` on Windows and `~/` on Linux don't work as expected when summoning the open file panel.
```
var paths = SFB.StandaloneFileBrowser.OpenFilePanel("Open File", "%userprofile%", extensions, false);
```
Using `%userprofile%` for example opens the panel on the same executing dir with `.../UnityApp/%userprofile%/` in the file name field.
![image](https://user-images.githubusercontent.com/14009559/156043540-0df372ab-a2ee-4123-b10e-6fd716f9eed5.png)
Using `C:/Users/user` instead corrects this behaviour to open on the user's home dir be default as desired, but its silly to hardcode user names.

**Comment by monsieurpooh** [comment]
Yes, I too was confused about this behavior. Is there any workaround?

**Comment by monsieurpooh** [comment]
WORKAROUND: %userprofile% can be derived by: `Environment.GetFolderPath(Environment.SpecialFolder.UserProfile)`
